<launch>
  <!-- Set `sim:=true` to launch a simulated 402 manager node -->
  <arg name="sim" default="false"/>

  <node
      name="hal_402_drives_mgr"
      pkg="hal_402_device_mgr"
      type="hal_402_drives_mgr.py"
      output="screen"
      required="true"
      >
    <env name="SIM" value="$(arg sim)"/>
  </node>
</launch>
